lib/utils.pm

```diff
@@ -226,21 +226,20 @@ Die, if this is not the case.
 
 =cut
 sub integration_services_check_ip {
-    # Workaround for poo#44771 "Can't call method "exec" on an undefined value"
-    select_console('svirt');
-    select_console('sut');
+    my ($keep_open) = shift // 1;
+
     # Host-side of Integration Services
     my $vmname = console('svirt')->name;
     my $ips_host_pov;
     if (check_var('VIRSH_VMM_FAMILY', 'hyperv')) {
         $ips_host_pov = console('svirt')->get_cmd_output(
-            'powershell -Command "Get-VM ' . $vmname . ' | Get-VMNetworkAdapter | Format-Table -HideTableHeaders IPAddresses"');
+            'powershell -Command "Get-VM ' . $vmname . ' | Get-VMNetworkAdapter | Format-Table -HideTableHeaders IPAddresses"', {keep_open => $keep_open});
```

Comment:
@b10n1k plz double check https://github.com/os-autoinst/os-autoinst/pull/1643/files 
the argument is only available for `run_cmd()` as `get_cmd_output()` is marked with **deprecated**. Maybe we need to do it there as well?
Or you could just use `run_cmd()` which I would prefer

```suggestion
            (undef, $ips_host_pov) = console('svirt')->get_cmd_output('powershell -Command "Get-VM ' . $vmname . ' | Get-VMNetworkAdapter | Format-Table -HideTableHeaders IPAddresses"', keep_open => 0, wantarray => 1);
```

Also if that `powershell` command do not need any "shell-context/environment" then I would do it just always.

Reply:
that was a very good catch. I replace the though `run_cmd` with `run_ssh_cmd` following the history as the later is wrapper of the `run_cmd`